src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/BuildIntegrationTest.cs

```diff
@@ -625,28 +625,26 @@ public async Task Build_WithoutServer_ErrorDuringBuild_DisplaysErrorInMsBuildOut
         {
             var result = await DotnetMSBuild(
                 "Build",
-                "/p:UseRazorBuildServer=false /p:RazorLangVersion=5.0",
+                "/p:UseRazorBuildServer=false /p:RazorLangVersion=99.0",
                 suppressBuildServer: true);
 
             Assert.BuildFailed(result);
             Assert.BuildOutputContainsLine(
                 result,
-                $"Invalid option 5.0 for Razor language version --version; must be Latest or a valid version in range 1.0 to 3.0.");
+                $"Invalid option 99.0 for Razor language version --version; must be Latest or a valid version in range 1.0 to 5.0.");
```

Comment:
> or a valid version in range 1.0 to 5.0.

This is a confusing statement indicating 4.0 might be valid. Could we rephrase this somehow?

Reply:
Suggestions? I would note that none of the point values are supported. (e.g. 3.1 is not valid).

Reply:
If it is easy to do, we could just list all the possible supported values. Just like how dotnet lists all the available SDKs

Reply:
Here's what the compiler does: 

Error	CS1617	Invalid option '9.0' for /langversion. Use '/langversion:?' to list supported values

It's not really easy to find this. I'm kinda tempted to leave these be. AFAIK, nobody configures these. Plus the language versions are precariously tied to the runtime version that setting this is always going to get you in trouble.